tests/conftest.py

```diff
@@ -9,22 +10,17 @@
 
 from xmipy import XmiWrapper
 
+from .common import libmf6_path
 
-@pytest.fixture(scope="session")
-def modflow_lib_path(tmp_path_factory):
-    tmp_path = tmp_path_factory.getbasetemp()
-    sysinfo = platform.system()
-    if sysinfo == "Windows":
-        lib_path = tmp_path / "libmf6.dll"
-    elif sysinfo == "Linux":
-        lib_path = tmp_path / "libmf6.so"
-    elif sysinfo == "Darwin":
-        lib_path = tmp_path / "libmf6.dylib"
-    else:
-        raise RuntimeError(f"system not supported: {sysinfo}")
 
-    flopy.utils.get_modflow(bindir=str(tmp_path))
-    return str(lib_path)
+def pytest_report_header(config, start_path, startdir):
```

Comment:
I couldn't find docs for this.
Could you add a link to the documentation for this functionality please?

Reply:
Added [link](https://docs.pytest.org/reference/reference.html#pytest.hookspec.pytest_report_header) to docstring.